sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1079,7 +1099,7 @@ object CodeGenerator extends Logging {
   /**
    * Records the generated class and method bytecode sizes by inspecting janino private fields.
```

Comment:
Better to update method comment if we change method purpose.

Reply:
oh, yea! Thanks! I'll update